apps/laboratory/src/pages/library/ethers.tsx

```diff
@@ -18,7 +18,8 @@ const modal = createWeb3Modal({
   termsConditionsUrl: 'https://walletconnect.com/terms',
   privacyPolicyUrl: 'https://walletconnect.com/privacy',
   customWallets: ConstantsUtil.CustomWallets,
-  enableOnramp: true
+  enableOnramp: true,
+  featuredWalletIds: ['c57ca95b47569778a828d19178114f4db188b89b763c899ba0be274e97267d96']
```

Comment:
Do we wan't to commit this?

Reply:
I wanted to use it to test this on lab preview.